lib/graphql/schema.rb

```diff
@@ -887,6 +889,14 @@ def max_complexity(max_complexity = nil)
         end
       end
 
+      def max_validation_errors(max_validation_errors = nil)
+        if max_validation_errors
+          @max_validation_errors = max_validation_errors
+        else
+          @max_validation_errors || find_inherited_value(:max_validation_errors) || 100
```

Comment:
Minor but `find_inherited_value` takes a default value as an optional second argument.

```suggestion
          @max_validation_errors || find_inherited_value(:max_validation_errors, 100)
```

Reply:
Hmm, didn't know that, thanks. I'm still trying to find a way to avoid duplicating the default value between here and the `initialize` method if you know how to do that.

Reply:
It's not very elegant but `Schema.to_graphql` could skip assigning `max_validation_errors` if it's null to avoid clobbering the defaulting done by the constructor i.e.

```ruby
schema_defn.max_validation_errors = max_validation_errors if max_validation_errors
```
Hopefully all of this gets cleaned up in 2.0 when the legacy config objects have been removed and there's a simple way to declare config options + their default values.

Reply:
I ended up just extracting it into a constant as the more obvious thing. And did move it into `find_inherited_value` too.